.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
Force to use workspace typescript version for vscode.

VSCode is very popular tool, but it always uses their own typescript sdk as default unless project has this setting.